frontend/src/components/Title/Title.tsx

```diff
@@ -2,17 +2,17 @@ import React from 'react'
 import { makeStyles } from '@material-ui/core'
 import { spacing, colors } from '../../styling'
 
-export const Title: React.FC<{ online?: boolean }> = ({ children, online }) => {
+export const Title: React.FC<{ offline?: boolean }> = ({ children, offline }) => {
```

Comment:
Found this component used without the online state were going grey. Not good so reversed the default.